tests/integration/targets/vmware_ops_esxi_maintenance_mode_test/vars/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+# General vars
+resource_prefix: host-maintenance 
+resource_pool_name: "{{ resource_prefix }}-pool"
+cluster_name: "Eco-Cluster"
+datacenter_name: "Eco-Datacenter"
+network_name: "TestNetwork"
+
+# Vars for cloud.vmware_ops.provision_virtual_esxi role
+provision_virtual_esxi_hostname: "{{ vcenter_hostname }}"
+provision_virtual_esxi_username: "{{ vcenter_username }}"
+provision_virtual_esxi_password: "{{ vcenter_password }}"
+provision_virtual_esxi_validate_certs: false
+provision_virtual_esxi_port: 443
+provision_virtual_esxi_cluster: "{{ cluster_name }}"
+provision_virtual_esxi_datacenter: "{{ datacenter_name }}"
+provision_virtual_esxi_folder: ""
+provision_virtual_esxi_resource_pool: "{{ resource_pool_name }}"
+provision_virtual_esxi_datastore_iso_path: "[eco-nfs-datastore-iso] custom_esxi_8.iso"
+provision_virtual_esxi_vms:
+  - name: ci-vcenter-maintenance-test
+provision_virtual_esxi_networks:
+    - name: "{{ network_name }}"
+      device_type: "vmxnet3"
+      type: "dhcp"
+      mac: "00:50:56:8b:93:a5"
+provision_virtual_esxi_disks:
+  - size_gb: 100
+    type: thin
+    datastore: datastore1
+
+# Vars for cloud.vmware_ops.vcenter_host_connection role
+vcenter_host_connection_hostname: "{{ vcenter_hostname }}"
+vcenter_host_connection_username: "{{ vcenter_username }}"
+vcenter_host_connection_password: "{{ vcenter_password }}"
+vcenter_host_connection_validate_certs: false
+vcenter_host_connection_port: 443
+vcenter_host_connection_cluster: "{{ cluster_name }}"
+vcenter_host_connection_datacenter: "{{ datacenter_name }}"
+vcenter_host_connection_esxi_username: "root"
+vcenter_host_connection_esxi_password: "!#%135qEt"
```

Comment:
@shellymiron is it ok to publish this? is this the same password of our hosts or is it used just for testing?

Reply:
theoretically it is ok, I'll explain and let me know what you think - when we first boot a new vm with esxi image, there is a manual step that needs to be done from the UI, which set up the password and storage etc...
those steps are not really possible to do using the playbook since there is no role that can detect the steps from within the the esxi GUI. the workaround that is done by @mikemorency is to create a kickstart bootable custom_esxi image, that has the password predefined before install, and attached to this ISO. see [here](https://github.com/mikemorency/vmware-playbooks/blob/main/playbooks/files/ks_cust.cfg) an example.
so what I eventually did, was to create this predefined iso image and upload it to the vcenter, so that now we can use the password to enter the esxi machine when needed. this password obviously can be changed and a new ISO can be generated with it, but this is the default one. so we can say this iso (with it's password) is used for testing purposes and not related to the hosts, the hosts has there own password we defined manually. 
wdyt?

Reply:
that's fine